Tasks/HelmDeployV0/src/helmcommands/helminstall.ts

```diff
@@ -26,17 +26,25 @@ export function addArguments(helmCli: helmcli): void {
     var updatedependency = tl.getBoolInput('updatedependency', false);
     var waitForExecution = tl.getBoolInput('waitForExecution', false);
     var argumentsInput = tl.getInput("arguments", false);
-    var valueFile = tl.getInput("valueFile", false);
+    var valueFilesInput = tl.getInput("valueFile", false);
```

Comment:
You could also use `getDelimitedInput` given that it's just a `\n` and not a regex

Reply:
I'm using both comma and newline as delimiters. That's why I didn't use `getDelimitedInput`

Reply:
Makes sense

Reply:
Add this info to the `helpText` of the `valueFile` input